src/core/config.go

```diff
@@ -108,6 +108,7 @@ func DefaultConfiguration() *Configuration {
 	config.Cache.Dir = ".plz-cache"
 	config.Cache.DirCacheHighWaterMark = "10G"
 	config.Cache.DirCacheLowWaterMark = "8G"
+	config.Metrics.PushFrequency = 100 // push every 100ms
```

Comment:
Seems like a good value, do we have the notion of pushing after the whole build has failed too (so if I set this high, it will still try to flush unpushed metrics before please quits?)